### DISCUSSION TITLE
[Feature] Import from Google Takeout Photo Dump

### DISCUSSION THREAD

**Discussion by ttufts** [immich-app/immich#10909] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [ ] Yes

### The feature

Google Photos allows you to export/download all photos/albums/data using their Takeout system. 

### Platform

- [x] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Dupe of #455. You can do this with https://github.com/simulot/immich-go
